config/locales/server.en.yml

```diff
@@ -19,6 +19,18 @@ en:
     ai_sentiment_inference_service_api_endpoint: "URL where the API is running for the sentiment module"
     ai_sentiment_inference_service_api_key: "API key for the sentiment API"
     ai_sentiment_models: "Models to use for inference. Sentiment classifies post on the positive/neutral/negative space. Emotion classifies on the anger/disgust/fear/joy/neutral/sadness/surprise space."
+
+    ai_openai_api_key: "API key for OpenAI API"
+
+    ai_embeddings_enabled: "Enable the embeddings module."
+    ai_embeddings_discourse_service_api_endpoint: "URL where the API is running for the embeddings module"
+    ai_embeddings_discourse_service_api_key: "API key for the embeddings API"
+    ai_embeddings_models: "Models to use for inference. `all-mpnet-base-v2` is used for suggested topics, `msmarco-distilbert-base-v4` for search and `text-embedding-ada-002` can do both."
```

Comment:
What happens with suggested topics when I set this setting to `smarco-distilbert-base-v4` for example?

Reply:
updated copy